packages/web/src/components/spa/Navigation/MobileNav/style.css

```diff
@@ -9,6 +9,31 @@
     color: white !important;
 }
 
+.navmobile-button-ham p {
+    padding: 0;
+    margin: 0;
+    font-size: xx-large;
+}
+
+.navmobile-button-ham {
+    width: 200px;
+    font-size: 16px;
+    font-weight: 600;
+    color: #fff;
+    cursor: pointer;
+    margin: 20px;
+    height: 55px;
+    text-align:center;
+    border: none;
```

Comment:
Do you need border `none` ? If not, remove it. Our goal is to ship the least amount of "bytes" and applicable styles from the server to the browser.